CONTRIBUTING.md

```diff
@@ -18,8 +18,6 @@ Before you open a PR:
   know. We'll help you get started, rather than inadvertently doubling up on your hard work.
 - Make sure you do not add regressions by running `yarn test`.
 - Where possible, include tests with your changes, either that demonstrates the
-  bug, or tests the new functionality. If you're not sure how to test your
-  changes, feel free to ping @gwyneplaine
-- Please [follow our established coding conventions](https://github.com/keystonejs/keystone/wiki/Coding-Standards)
-  (with regards to formatting, etc)
+  bug, or tests the new functionality.
+- Please [follow our contributors guidelines](https://github.com/JedWatson/react-select/blob/master/CONTRIBUTING.md).
```

Comment:
Is there any point this being a link if it just links to the current document?

Reply:
No, let's remove the line completely